python/ray/_private/runtime_env/install_ray_or_pip_packages.py

```diff
@@ -18,7 +18,7 @@ def install_ray_package(ray_version, whl_dir):
     if whl_dir:
         # generate whl file name
         whl_file_name = (
-            f"ant_ray-*cp{sys.version_info[0]}" f"{sys.version_info[1]}*.whl"
+            f"ant_ray-*cp{sys.version_info.major}{sys.version_info.minor}*.whl[default]"
```

Comment:
This means the wheel file has to be ended with ".whl[default]", which seems weird. Can we match the wheel file  name with "ant_ray-*cp*.whl", but install it by the specific wheel file name with suffix "[default]"?